students/arkadiusz_kasprzyk/lesson_04_unit_testing/test_collatz.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+
+
+def collatz(number):
+    if number % 2 == 0:
+        return number // 2
+    else:
+        return 3 * number + 1
+
+
+class TestCollatz(unittest.TestCase):
+
+    def test_type(self):
```

Comment:
Try to make a name a statement about the code `test_collatz_raises_TypeError_on_aoeu`, Dvorak bro! This way when the test fails you can jump to code directly.